.github/workflows/docs.yml

```
@@ -15,15 +15,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-
-      - uses: actions/setup-node@v4
-        with:
-          node-version: 20.x
+      - uses: ./.github/actions/ci-setup
 
       - name: Build docs
         working-directory: docs
         run: |
-          yarn install
           NODE_OPTIONS='--openssl-legacy-provider' yarn docs:build
```

Should this be this?

```suggestion
          NODE_OPTIONS='--openssl-legacy-provider' pnpm docs:build
```